### ISSUE TITLE
jack server quits after it can't find tmpdir

### ISSUE THREAD

**Issue by lsiden** [jackaudio/jack1#105]
I installed qJackctl and jack with Brew.  Then I started it from the command line:  "jack -d coreaudio".
Next I start qJackctl.  jack exits:

*****
jackd -d coreaudio
jackd 0.125.0
Copyright 2001-2009 Paul Davis, Stephane Letz, Jack O'Quinn, Torben Hohn and others.
jackd comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details

JACK compiled with POSIX SHM support.
loading driver ..
Error calling AudioUnitSetProperty - kAudioUnitProperty_StreamFormat kAudioUnitScope_Output
jack main caught signal 15
poll failed (Bad file descriptor)
*****

For the record:
*****
jackd -v
jackd 0.125.0
Copyright 2001-2009 Paul Davis, Stephane Letz, Jack O'Quinn, Torben Hohn and others.
jackd comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details

jackd 0.125.0
Copyright 2001-2009 Paul Davis, Stephane Letz, Jack O'Quinn, Torben Hohn and others.
jackd comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details

uname -a
Darwin neo.lan 19.6.0 Darwin Kernel Version 19.6.0: Mon Aug 31 22:12:52 PDT 2020; root:xnu-6153.141.2~1/RELEASE_X86_64 x86_64
*****

Then I downloaded the source-code and tried to build it:

******
 ./configure
configure: error: cannot find install-sh, install.sh, or shtool in config "."/config
******

For the record:
******
git show =>
commit a4e2355c964b4ec518f2e870667548748e8747f9 (HEAD -> master, origin/master, origin/HEAD)
******

So the brew release won't run, and I can't build it myself. 
Any suggestions how to proceed?

**Comment by falkTX** [comment]
yes, use the releases at https://github.com/jackaudio/jack2-releases/releases
